0-data-roles/0-one-file/src/handlers.js

```diff
@@ -0,0 +1,20 @@
+import { INITIAL_STATE, EVENTS, HTML_CLASSES } from './constants.js';
+const handlers = (EVENTS) => {
```

Comment:
The name of the arguments must not be UPPERCASED.

```suggestion
const handlers = (events) => {
```